modules/tts/tts-adapters/tts-adapter-sapinative/src/main/java/org/daisy/pipeline/tts/sapi/impl/SAPIEngine.java

```diff
@@ -17,15 +17,10 @@
 import net.sf.saxon.s9api.XdmNode;
 
 import org.daisy.common.file.URLs;
-import org.daisy.pipeline.tts.onecore.Onecore;
-import org.daisy.pipeline.tts.onecore.OnecoreResult;
-import org.daisy.pipeline.tts.onecore.SAPI;
-import org.daisy.pipeline.tts.onecore.SAPIResult;
+import org.daisy.pipeline.tts.onecore.*;
```

Comment:
I prefer imports of classes or static methods, not whole packages.

Reply:
Intellij tends to rewrote import this way, i'll update that

Reply:
done

Reply:
Intellij does this? Hmm I see.